src/devices/sound/ics2115.cpp

```diff
@@ -30,6 +25,8 @@
 
 //#define ICS2115_DEBUG
 //#define ICS2115_ISOLATE 6
+const static int PAN_LEVEL = 16;
+const static int RAMP_SHIFT = 6;
 
```

Comment:
Are these 2 configuration settings, or magic number clarifications?
If latter, they can be made local since they're only used at one place.